.travis.yml

```diff
@@ -30,30 +26,30 @@ addons:
       - g++-4.8
 
 before_install:
-- npm install -g npm@latest-2
-- npm --version
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.3.2
+  - export PATH=$HOME/.yarn/bin:$PATH
+cache:
+  yarn: true
 
 install:
-# Remove any unncessary packages. Probably unnecessary but potentially helpful
-- npm prune
 # Update runs install, but will also pull in the latest updates so the cache
 # doesn't get stale
-- npm update
+- yarn upgrade
```

Comment:
Let's not keep this step - it's not relevant with yarn usage and locked dependencies, and in fact might cause problems. It was originally there to catch the case where npm deps would get updates and our dependency ranges would result in breakages that we didn't see. Locks prevent that.